Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.yml

```diff
@@ -13,6 +13,7 @@ configuration:
   options:
   - OpenLDAP
   - Active Directory
+  - Auto
```

Comment:
Please add a comment in the readme that using `Auto` will try to determine it automatically, and let's also change the `default` to `Auto`.